docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md

```diff
@@ -0,0 +1,147 @@
+---
+id: air-gapped-installation
+title: "Installing in an air-gapped environment"
+description: "Camunda Platform 8 Self-Managed installation in an air-gapped environment"
+---
+
+The [Camunda Platform Helm chart](../../helm-kubernetes/deploy.md) may assist in an air-gapped environment. By default, the images are fetched via Docker Hub.
+With the dependencies in third-party images and Helm charts, additional steps are required to make all charts available as outlined in this resource.
+
+## Required images
```

Comment:
Let's call it `Required Docker images` for clearness and consistency (as we have "Required Helm charts").